docs/handling-uuids-on-post-put.md

```diff
@@ -0,0 +1,51 @@
+# Handling UUIDs in `POST` and `PUT` operations
+
+## `PUT` vs `POST`
+
+```mermaid
+graph LR
+  Init([Make a request]) --> Pref{Preference on UUID}
+  Pref -->|Yes| PUT
+  Pref -->|No| POST
+```
+
+## `POST`
+
+When creating new objects, `POST` is the preferred option. The UUID of the new object
+does not need to be specified, so `POST` should be used when there is no UUID preference.
+If the UUID is specified in the body, an HTTP 4xx error will be returned.
```

Comment:
Isn't the error now specified by #54?

Reply:
I'm not sure if 409 would be the response to the UUIDs not matching. It is returned when there is a request conflict with the current state (the most common example being an issue with versioning) so I'm not sure if mismatched UUIDs would fall under this or not

Reply:
Ok that makes sense. If it is not a `409` error, is there one more applicable? I don't know if we want just `4xx`

Reply:
Mismatched UUIDs is what we're talking about primarily at the moment with `409 Conflict` status code. That is the primary conflict in play